src/main/java/org/folio/rest/service/action/ActionService.java

```diff
@@ -159,6 +164,15 @@ protected Feefineaction createFeeFineActionAndUpdateAccount(Account account, Mon
     return feeFineAction;
   }
 
+  private void populateMetadata(Account account) {
+    try {
+      MetadataUtil.populateMetadata(account, headers);
+    } catch (ReflectiveOperationException e) {
+      log.error("Can not populate Metadata, cause {}", e.getMessage());
+      account.getMetadata().setUpdatedDate(new Date());
```

Comment:
I think it is possible that metadata is null. For example, if an account was created not from a JSON-request, but manually from the code. So please take necessary precautions.